src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -86,129 +91,119 @@ export default class AdvancedFilters extends Vue {
     );
   }
 
-  _createInputFilter(filter: DataTableFilter, index: number) {
+  async _changeFormElementFilter(ev: Event, elementType: DataTableFormElementFilters) {
+    if (ev.target) {
+      const newFilterData: {
+        name: string;
+        checked?: boolean;
+        value?: string;
+        id: string;
+      } | null = getElementFilterData(ev, elementType);
+
+      await this.storeModule.updateFilterConfig({
+        ...this.filterElementValue,
+        ...{
+          [newFilterData?.id.replace(/-desktop|-mobile/gi, '') || 'no-id']:
+            elementType !== 'checkbox' ? newFilterData?.value : newFilterData?.checked,
+        },
+      });
+
+      if (newFilterData && this.advancedFiltersData) {
+        updateFilterData(this.advancedFiltersData, newFilterData);
+      }
+    }
+  }
+
+  _createInputFilter(filter: DataTableFilter) {
     return (
       <div
         id={this._calculateFilterId(filter.name)}
         // Todo
         class={`
-          ${FORM_CLASSES.FORM_ITEM}
-          ${index !== 0 && !filter.advanced ? 'chi-data-table-filter__input -ml--2' : ''}`}>
+              ${FORM_CLASSES.FORM_ITEM}`}>
+        {this.mobile && (
+          <label for={filter.id} class={FORM_CLASSES.LABEL}>
+            {filter.name}
```

Comment:
```suggestion
            {filter.label}
```
If you check out the screenshot above, you will see that it renders the filter name. Initially, filter name served as a unique ID of the filter. That's why it's in uppercase. Please, use `filter.label`instead